src/modules/package.ts

```diff
@@ -149,3 +143,32 @@ export function isPackageInstalled(packageName: string, versionIdentifier: strin
     }
     return true;
 }
+
+async function downloadPackageRequest(packageName: string, versionIdentifier: string): Promise<AxiosResponse<any, any>> {
+    const defaultUrl = `${PACKAGE_REPO(packageName)}/zipball/refs/tags/${versionIdentifier}`;
+    const fallbackUrl = `${PACKAGE_REPO(packageName)}/zipball/${versionIdentifier}`;
+    const requestHeaders: AxiosRequestHeaders = {
+        accept: 'application/vnd.github+json',
+    };
+    setApiToken(requestHeaders);
+
+    axiosRetry(axios, {
+        retries: 3,
+        retryCondition: (error) => {
+            if (error.response?.status === 404) {
+                console.log(`Did not find tag '${versionIdentifier}' in '${packageName}' - looking for branch ...`);
```

Comment:
Please hide this behind a `verbose` flag. It is too much information for the ordinary user.
`velocitas init -v`